.idea/electron-builder.iml

```diff
@@ -6,11 +6,11 @@
       <excludeFolder url="file://$MODULE_DIR$/example-app" />
       <excludeFolder url="file://$MODULE_DIR$/node_modules" />
       <excludeFolder url="file://$MODULE_DIR$/out" />
+      <excludeFolder url="file://$MODULE_DIR$/test/_published" />
```

Comment:
Otherwise IDE completion tries to use fake package.json — so, just move to separate directory.